test_cvm/test-game.js

```
@@ -397,6 +512,37 @@ async function runTest() {
       await new Promise(resolve => setTimeout(resolve, 1000));
     }
     
+    // Step 8: Post small blind for the player in the small blind position
+    if (gameState && gameState.data && gameState.data.players) {
+      // Find small blind player
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)